app/routes/provider.js

```diff
@@ -0,0 +1,44 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+const providers = config.Reviews.providers.slice(1);
+const providerIds = providers.map(p => p.id);
+
+/**
+ * @module ember-osf-reviews
+ * @submodule routes
+ */
+
+/**
+ * @class Provider Route Handler
+ */
+export default Ember.Route.extend({
+    theme: Ember.inject.service(),
+
+    beforeModel(transition) {
+        const {slug = ''} = transition.params.provider;
+        const slugLower = slug.toLowerCase();
+
+        if (providerIds.includes(slugLower)) {
```

Comment:
@aaxelb I removed some of the logic and kept one to check for provider id. I added also a **todo** note to remove in future